data/json/professions.json

```diff
@@ -2389,14 +2405,22 @@
         "ident": "bionic_spy",
         "name": "Bionic Agent",
         "description": "Your body has several bionics worth millions of dollars, paid for by public taxes.  The government has turned you into an infiltration and recon specialist: you have night vision, an alarm and a hacking module.  And it all runs off piles and piles of everyday household batteries.",
-        "points": 3,
+        "points": 6,
```

Comment:
It's a really horrible profession for 6 points. It has 2 good bionics: batteries and storage. Nightvision could be useful in a pinch, but without a reliable way to recharge energy, it's just too expensive to keep it up.
Fingerhack, alarm, face mask, lockpick and speech are all situational and not all that useful except for recruiting hordes of NPCs.